src/en/space-station-14/character-species/proposals/species-languages.md

```diff
@@ -0,0 +1,182 @@
+# Species Languages
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| TheNat'LaGuy | :x: No | TBD |
+
+## Overview
+
+This Proposal aims to add in-game languages based on what species you are currently playing, by giving the player the ability to speak in their species native languages that only other members of the same species can understand (With some cavaets depeding on what species you play and even your job)
+
+## Design Pillars
+
+### The galatic standard language
+
+Every single member of the crew (and sentient beings, made sentient by an event or cognize), by default, speaks the Galactic Common Language, the equivelent of present day english. Any word you say by default while whispering, talking and speaking on the radio is automatically said in Galactic Common.
+
+Nonsentient beings, like space carp, smile, hamlet and any ghost role or creature can understand Galactic Common, but can't speak it.
```

Comment:
> 
> if any being can understand galatic common, what's the point of a slime person not being able to speak slime?
> 
> since they can understand the slime, and the slime can understand galatic common, they can already communicate with eachother
> 
> 

I'd like this to be addressed

Reply:
I've already responded!

> Although Slimes can understand galactic common, Normal people, except Slime people and the Librarian, cant, so besides emotes and poiting, they cant communicate through open dialogue
> 
> Think of it this way: If you have a dog, they can (somewhat) understand what your saying, through proper traning of course, but you can't speak "dog", so you have to resort to what their body language is saying to see if they need anything, like pets or food. Now imagine, that trough the infinite power of the universe, you somehow understood "dog". Not speak "dog", just understand it. Their barks start sounding like actual words, like "I want to eat beef!" and similiar. Even though your dog cant speak english, and you can't speak "dog", you can understand it, which makes life easier for you as an owner and your dog.

Reply:
I've already responded!

https://github.com/space-wizards/docs/pull/298#issuecomment-2327816378

Reply:
didn't see sorre